docs/deploy_lowcost.md

```diff
@@ -0,0 +1,69 @@
+# Deploying with minimal costs
+
+This AI RAG chat application is designed to be easily deployed using the Azure Developer CLI, which provisions the infrastructure according to the Bicep files in the `infra` folder. Those files describe each of the Azure resources needed, and configures their SKU (pricing tier) and other parameters. Many Azure services offer a free tier, but the infrastructure files in this project do *not* default to the free tier as there are often limitations in that tier.
+
+However, if your goal is to minimize costs while prototyping your application, follow these steps below _before_ deploying the application.
+
+1. Use the free tier of App Service:
+
+    ```shell
+    azd env set AZURE_APP_SERVICE_SKU F1
+    ```
+
+    Limitation: You are only allowed a certain number of free App Service instances per region. If you have exceeded your limit in a region, you will get an error during the provisioning stage. If that happens, you can run `azd down`, then `azd env new` to create a new environment with a new region.
+
+2. Use the free tier of Azure AI Search:
+
+    ```shell
+    azd env set AZURE_SEARCH_SERVICE_SKU free
+    ```
+
+    Limitations:
+    1. You are only allowed one free search service across all regions.
+    If you have one already, either delete that service or follow instructions to
+    reuse your [existing search service](../README.md#existing-azure-ai-search-resource).
+    2. The free tier does not support semantic ranker, so you will need to disable
```

Comment:
Is it possible to check the env var in the client and disable semantic ranker if the search service tier is free?

Reply:
Yes, I can do that, in progress. Just need to make sure it'll play nicely with existing deployments.